src/components/Chart/data/dataSelector.js

```diff
@@ -83,7 +84,7 @@ export default () => ({
             for (let i = 0; i < numberOfBits; ++i) {
                 const y = Number.isNaN(v)
                     ? undefined
-                    : (((bits[k] >> i) & 1) - 0.5) * 0.8;
+                    : [-0.5, -0.4, 0.4, 0][doubleBitValue(bits[k], i)];
```

Comment:
I can't really tell what is going on here :D Can we extract this to a function and give it a more descriptive name to help me and others like me? :) I don't know what these magic numbers (-0.5, -0.4 etc) refers to at all

Reply:
Best not to bother with that now, as I expect it will change with Marko's new double line and filled area representation. However the values are a the up and down state of the bit in [-0.5, 0.5] range with a slight margin, hence 0.4.